thirdparty/yaml-cpp-yaml-cpp-20171024/test/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+set(gtest_force_shared_crt ${MSVC_SHARED_RT} CACHE BOOL
```

Comment:
Can we kill the test folder.  We had done this previously and will help keep our LICENSE a little tidier.